python/ray/serve/scripts.py

```diff
@@ -353,18 +295,16 @@ def deploy(config_file_name: str, address: str):
         "Directory containing files that your job will run in. Can be a "
         "local directory or a remote URI to a .zip file (S3, GS, HTTP). "
         "This overrides the working_dir in --runtime-env if both are "
-        "specified. Overrides all working_dirs specified in a config file."
+        "specified. Overrides working_dirs specified in the config file."
     ),
 )
 @click.option(
     "--address",
     "-a",
-    default="auto",
+    default=os.environ.get("RAY_ADDRESS", None),
```

Comment:
Will this pass the tests in `test_cli.py`? We're not passing in an address there.

Reply:
probably not, let me fix

Reply:
passed locally

Reply:
what's the reason for not defaulting to "auto" here?

Reply:
For `serve run`, if Ray isn't running we just want to call `ray.init()`